models/prices/solana/prices_solana_tokens.sql

```diff
@@ -11,41 +11,661 @@ SELECT
+        ('hawk-hawksight','solana','$HAWK','BKipkearSqAUdNKa1WDstvcMjoPsSKBuNyvKDQDDu9WE',6),
+        ('popo-popo-pepe39s-dog','solana','$POPO','96QoNkZLTKH7Gecf6dKMRXBQXW1wqh378uF9pnTejgAw',9),
+        ('acs-access-protocol','solana','ACS','5MAYDfq5yxtudAhtfyuMBuHZjgAbaS9tbEyEQYAhDS5y',6),
+        ('peow-felix-the-lazer-cat','solana','$PEOW','7DptUgc1aRY6geK4WnauRKTTfVFWmLsBWP275sD2Bg9r',9),
+        ('pigeon-pigeon-in-yellow-boots','solana','PIGEON','Gv7sW9pe9khuPjCYmRUTmHNAFm8yWDJVtDVyA1DExqr4',6),
+        ('saros-saros','solana','SAROS','SarosY6Vscao718M4A778z4CGtvcwcGef5M9MEH1LGL',6),
+        ('sbonk-shibonk','solana','SBONK','H1G6sZ1WDoMmMCFqBKAbg9gkQPCo1sKQtaJWz9dHmqZr',9),
+        ('knk-kineko','solana','KNK','kNkT1RDnexWqYP3EYGyWv5ZtazB8CfgGAfJtv9AQ3kz',9),
+        ('why-wenwifhat','solana','WHY','WHYoaBumcmxCqw38y2mjs4cVkCBgwiDizbMVvcejmGT',9),
+        ('spooky-spooky-the-phantom','solana','SPOOKY','FdGoS1Dok5CLnS8fVSmj5A92uY1yhzdTC2ZxuLJdkwgs',6),
+        ('bonk-bonk','solana','BONK','DezXAZ8z7PnrnRJjz3wXBoRgixCa6xjnB7YaB1pPB263',5),
+        ('snakes-snakes-game','solana','SNAKES','FuvFLtx68uGjhr4GWET8JTskC7s3R7MM4Feg8toyzFvf',6),
+        ('dxl-dexlab','solana','DXL','GsNzxJfFn6zQdJGeYsupJWzUAm57Ba7335mfhWvFiE9Z',6),
+        ('cheems-cheems','solana','CHEEMS','3FoUAsGDbvTD6YZ4wVKJgTB76onJUKz7GPEBNiR5b8wc',4),
+        ('andy-andy-sol','solana','ANDY','93n51i8anR3ziS6kxqXEACjrvJoXpAxt3qbbo3eaRFRz',9),
+        ('audio-audius','solana','AUDIO','9LzCMqDgTKYz9Drzqnpgee3SGa89up3a247ypMj2xrqM',8),
+        ('lfg-lessfngas','solana','LFG','LFG1ezantSY2LPX8jRz2qa31pPEhpwN9msFDzZw4T9Q',7),
+        ('gogo-gogowifcone','solana','GOGO','Byb1W95xDYBiF6gMJoTRgpD7Z14GFna2QLHoZfEWNbfQ',9),
+        ('bobo-bobo-on-sol','solana','BOBO','ArRRiz4XsPbLXNCUWb2dqE85hPxD4MtMhec5jaJMcW5R',5),
+        ('frog-forrealog','solana','FROG','HZ32SiTtw3kYyaHTtTfpHVF8EyXFcy7MBQXeFpnNvQ9c',2),
+        ('fronk-fronk','solana','FRONK','5yxNbU8DgYJZNi3mPD9rs4XLh9ckXrhPjJ5VCujUWg5H',5),
+        ('pepe-pepecoin-on-sol','solana','PEPE','3tS6fbLh2P8tzxXuqCiHZpZhsxJpmrR3Xb9psmypnp69',9),
+        ('peep-peepo','solana','PEEP','n54ZwXEcLnc3o7zK48nhrLV4KTU5wWD4iq7Gvdt5tik',6),
+        ('dragon-dragoncoin','solana','DRAGON','GiBrdw1tF8nuJxWuhTp83ULEMY9uJkYUHQUBzwfEnw5R',0),
+        ('pipi-pipi-the-cat','solana','PIPI','BZKuDqzD4rb2puUV2cbSNd2kZzJ5JzKiQKwBzCk8hrLu',9),
+        ('elmina-elmina','solana','CHUCHU','ELST1oAp3czkRRRWGeTgttGsEqkD2qjDRJXR4Asmkk5y',6),
+        ('chipi-chipi','solana','CHIPI','chiPiQTvkQ7oPtAD7YLQaEeHmPqXCa2wcRQdwFNneTe',9),
+        ('speero-speero','solana','SPEERO','C7K4Tot6fnnNwhWpqw9H277QPcP56vHAEeXubRHDyCo9',6),
+        ('burrrd-burrrd','solana','BURRRD','F8qtcT3qnwQ24CHksuRrSELtm5k9ob8j64xAzj3JjsMs',4),
+        ('ewon-ewon-mucks','solana','EWON','252ezuEq49e24uwSdEt16WoTc4Fw4SQLwWdhvH847Uer',7),
+        ('eth-ethereum','solana','ETH','7vfCXTUXx5WJV5JADk17DUJ4ksgau7utNKj4b963voxs',8),
```

Comment:
This line is duplicated with Line37 `weth`:
https://github.com/duneanalytics/spellbook/pull/5711/files#diff-a04317ac77e907d87f61b30de96ba9dea282120846886e1a99af70a3ca7821e9R37

We may only keep one of them.